core/base/contourTreeAlignment/ContourTreeAlignment.cpp

```diff
@@ -1220,9 +1220,8 @@ std::shared_ptr<AlignmentTree> ttk::ContourTreeAlignment::traceAlignmentTree(
   if(t2 == nullptr)
     return traceNullAlignment(t1, true);
 
-  auto id = [this](std::shared_ptr<BinaryTree> t) {
-    return t == nullptr ? 0 : t->id;
-  };
+  auto id
```

Comment:
@floWetzels 
can you please confirm you're ok with the edits made to this class?